src/index.tsx

```diff
@@ -1,24 +1,26 @@
-import React, { useState } from "react";
-import ReactDOM from "react-dom";
 import * as serviceWorker from "./serviceWorker";
-import { GlobalStyle } from "shared/styles";
-import { signUpConfig, amplifyConfig } from "shared/amplify.config";
-import { isAuthenticated, Config } from "shared/utils";
-import { Authenticator, UsernameAttributes } from "aws-amplify-react";
-import "@aws-amplify/ui/dist/style.css";
+import React from "react";
+import ReactDOM from "react-dom";
 import App from "App";
+import { GlobalStyle } from "shared/styles";
+import { Authenticator } from "aws-amplify-react";
+import { Config, isAuthenticated } from "shared/utils";
+import { amplifyConfig, authenticatorConfig } from "shared/amplify.config";
+import { AuthState } from "shared/interfaces/amplify.interface";
```

Comment:
We should order imports that way:

- React 
- external libraries
- local import